specs/pac/pac-gitlab.spec

```diff
@@ -0,0 +1,17 @@
+PIPELINES-29
+# Pipelines As Code tests
+
+## Enable/Disable PAC: PIPELINES-29-TC01
```

Comment:
There is no "disable" part :) I would go with something generic and more descriptive like "Configure PAC in new project"